scripts/migrate/20190410-NS-445/copy_rds_metadata_from_redshift.sql

```diff
@@ -0,0 +1,125 @@
+-- This script needs the "redshift_db" variable set before being run.
```

Comment:
This seemed a bit more in line with our release conventions than writing a shell script, but let me know if you disagree.

Reply:
I agree.